issue_template.md

```diff
@@ -0,0 +1,24 @@
+### **Description**
+
+_Please describe the issue you are facing or a feature you would like to be added to the SDK._
+
+### **Repro Steps**
+
+_Please list the steps used to reproduce your issue._
+
+1.
+2.
+
+### **Details**
+
+1. Which app platform you are developing for?
+    - _Android/Apple/iOS_
+2. Which OS version did you experience the issue on?
+    - _e.g. iOS 11, Android 8.1.0_
+3. What device version did you see this error on?  Were you using an emulator or a physical device?
+    - _e.g. iPhone X physical device, Google Pixel 2 emulator_
+4. What third party libraries are you using? <!-- For .NET, you can find these in your .csproj, package.json, or package.config files -->
+    - _example_
+5. Please enable verbose logging for your app using the line below and include the logs here.
```

Comment:
We should say that enable verbose logging by using the API below right before AppCenter.start() API